packages/components/src/SidePanel/SidePanel.component.js

```diff
@@ -18,13 +18,16 @@ import theme from './SidePanel.scss';
  * @return {string}        	formatted id
  */
 function getActionId(id, action) {
-	const actionId =
-		action.id ||
-		action.label
-			.toLowerCase()
-			.split(' ')
-			.join('-');
-	return id && `${id}-nav-${actionId}`;
+	if (action.id || action.label) {
+		const actionId =
+			action.id ||
+			action.label
```

Comment:
I don't think we should force an id. If the user wants an id, it provides it.
Any reason or other opinion ?